.github/ISSUE_TEMPLATE/issue.yml

```diff
@@ -0,0 +1,42 @@
+name: Feature Request
```

Comment:
NIT: I'd recommend to keep the naming consistent between the yml file and the name specified in the template
Renaming the `issue.yml` file to `feature_request.yml` would be more explicit

Reply:
Additionally, you can also specify in what order the issue templates are shown by adding a numbering before the issue file name, which could make sense here as well (e.g. `01-bug-report.yml` and `02-feature-request.yml`) 🤔 
See [here](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#changing-the-order-of-templates)